cmd/keeper/keeper.go

```diff
@@ -1160,5 +1160,7 @@ func keeper(cmd *cobra.Command, args []string) {
 	}
 	go p.Start()
 
-	<-end
+	if err := <-end; err != nil {
```

Comment:
From what I understood, values sent from channels should never be `nil` or they'll block the flow indefinitely (as per https://golang.org/ref/spec#Operators)

So in theory if you get err value from `<-end` that should never be `nil`... from what I got :)